### PR TITLE
Performance improvements for isQuantifier

### PR DESCRIPTION
**Pull request by morwen**
I created benchmark, please take look  https://github.com/bd82/regexp-to-ast/issues/17

### CODE REVIEW

File: benchmark/benchmark.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const Benchmark = require("benchmark")
+const _ = require("lodash")
+const samples = require("./samples")
+const PublishedRegExpParser = require("regexp-to-ast").RegExpParser
```
Comment (bd82) [inline benchmark/benchmark.js]: does this gets resolved globally?

File: benchmark/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "version": "0.0.1",
+    "description": "Benchmark for regexp-to-asp",
+    "keywords": [
+        "parser"
+    ],
+    "main": "benchmark.js",
+    "repository": "https://github.com/bd82/regexp-to-ast.git",
+    "license": "MIT",
+    "dependencies": {
+        "benchmark": "^2.1.4",
```
Comment (bd82) [inline benchmark/package.json]: benchmark should be a dev dependency

File: benchmark/package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (bd82) [inline benchmark/package.json]: strange to have another package.json for the benchmark without a proper mono-repo structure

**Comment by bd82** [comment]
o.k. this is very interesting.
It looks as though using a try/catch in a productive flow to signal failure of backtracking.
but **not** total failure of parsing results in almost 2 orders of magnitude performance hit.

This must be some edge case V8 is not optimizing

**Comment by morwen** [comment]
Thank you, when should we expect this changes to be released to chevrotain?

**Comment by bd82** [comment]
Today

**Pull request merged**